### ISSUE TITLE
Doc warning in plot utils: Example->Examples

### ISSUE THREAD

**Issue by balarsen** [spacepy/spacepy#405]
Super minor but the section is `Examples` not `Example` in numpydoc. 

```
/home/balarsen/miniconda3/envs/python3/lib/python3.7/site-packages/numpydoc/docscrape.py:418: UserWarning: Unknown section Example in the docstring of <function add_arrows at 0x7f66696ea320> in /mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py.
  warn(msg)
/home/balarsen/miniconda3/envs/python3/lib/python3.7/site-packages/numpydoc/docscrape.py:418: UserWarning: Unknown section Example in the docstring of <function add_arrows at 0x7f66696ea320> in /mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py.
  warn(msg)
/home/balarsen/miniconda3/envs/python3/lib/python3.7/site-packages/numpydoc/docscrape.py:418: UserWarning: Unknown section Example in the docstring of <function add_arrows at 0x7f66696ea320> in /mnt/c/Users/234728/git/spacepy/spacepy/plot/utils.py.
  warn(msg)
```

spacepy github f56b6d6926b989ecdfcb7f3745eb2f9f5a51848d

**Comment by balarsen** [comment]
This was closed in #406

**Comment by jtniehof** [comment]
The "closes xxx" needs to be in the PR description not the comments, which is why this was left open.
